### PR TITLE
30 create oih delete action

### PR DESCRIPTION
**Pull request by ElBabbler**
Add Delete file action

### CODE REVIEW

File: component.json [hunk 1]
```diff
@@ -91,6 +91,41 @@
           }
         }
       }
+    },
+    "delete": {
+      "main": "./lib/actions/delete.js",
+      "title": "Delete files",
```
Comment (kirill-levitskiy) [inline component.json]: Please, rename the Action to `Delete File` if it removes only one file for one execution

File: component.json [hunk 2]
```diff
@@ -91,6 +91,41 @@
...
+      "description": "Delete file by name in provided package",
```
Comment (kirill-levitskiy) [inline component.json]: What `package` means in terms of the SFTP? Maybe better to replace it with `Delete file by path and filename`?

File: component.json [hunk 3]
```diff
@@ -91,6 +91,41 @@
...
+      "fields": {
+        "directory": {
+          "viewClass": "TextFieldView",
+          "label": "Directory Name",
```
Comment (kirill-levitskiy) [inline component.json]: How about replacing `Directory Name` with `Path`?

File: package.json [hunk 4]
```diff
@@ -15,6 +15,7 @@
   "license": "BSD-2-Clause",
   "dependencies": {
     "@elastic.io/component-commons-library": "0.0.6",
+    "@elastic.io/oih-standard-library": "^1.0.0",
```
Comment (kirill-levitskiy) [inline package.json]: Is `^` sign applicable to our standards for `package.json`?

File: component.json (hunk 3 above)
Comment (ElBabbler) [reply component.json]: Why - it's directory name?

**Pull request merged**